ocs_ci/utility/utils.py

```diff
@@ -724,6 +724,44 @@ def get_openshift_client(
     return client_binary_path
 
 
+def get_vault_cli(bind_dir=None, force_download=False):
+    """
+    Download vault based on platform
+    basically for CLI purpose. Binary will be directly
+    put into ocs_ci/bin/ directory
```

Comment:
Please add docstring for args.
Also it would be good to use some toolbox pod for cli operations instead of editing host system but there are no destructive operations so I think it's ok.

Reply:
+1 for adding docstrings for args eg: VAULT_SKIP_VERIFY and similar